single-node/user-data

```diff
@@ -83,7 +83,7 @@ function init_templates {
         mkdir -p $(dirname $TEMPLATE)
         cat << EOF > $TEMPLATE
 [Service]
-ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
```

Comment:
We should not move the static pod location (and they're not what you would consider addons).

`/etc/kubernetes/manifests` = static pods
`/srv/kubrnetes/addons` = addons
`/srv/kubernetes/manifests` = this has historically been used as a "at rest" location for inactive manifests.